oaknational/Oak-Web-Application
# Fix/1033 search

## Description

- Search term is now put in url query param `term`

## Issue(s)

Fixes #1033 

## How to test

1. Go to /beta/search
2. Add a `?term=[whatever you wanna search for]`
3. You should see results listed
4. Select key stage filters to results change

Note -  you can also uncomment the search field in the AppHeader to test with a form - the page should navigate from anywhere in beta to the search page on form submission.


## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
